<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1364</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="pbModule_1">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>80</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: rgba(44, 127,255, 0.4);
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: beige;
     font: bold 14px;
     min-width: 10em;
     padding: 6px;
}

QPushButton:hover {
     background-color: rgba(44, 127,255, 0.6);
}</string>
    </property>
    <property name="text">
     <string>Угадай художника по картине</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbModule_2">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>490</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: rgba(44, 127,255, 0.4);
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: beige;
     font: bold 14px;
     min-width: 10em;
     padding: 6px;
}

QPushButton:hover {
     background-color: rgba(44, 127,255, 0.6);
}</string>
    </property>
    <property name="text">
     <string>Ответь на вопросы</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCurrentStage">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>10</y>
      <width>151</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: orange;
}</string>
    </property>
    <property name="text">
     <string>0 / 20</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbResult">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>511</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
     background-color: rgba(44, 127,255, 0.4);
     border-color: beige;
	border-style: outset;
	border-width: 2px;
     border-radius: 10px;
     font: bold 11px;
     min-width: 10em;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="lblCorrect">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>57</y>
       <width>321</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Количество правильных ответов:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lclCorrect">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>121</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="mode">
      <enum>QLCDNumber::Dec</enum>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>20.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblQuestion">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>851</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
     background-color: rgba(167, 167, 167, 0.7);
     border-color: beige;
	border-style: outset;
	border-width: 2px;
     border-radius: 10px;
     font: bold 25px;
     min-width: 10em;
	color:  #21557c;
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblImage">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>620</y>
      <width>451</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblLogo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>491</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/лого.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbExit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>760</y>
      <width>84</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: rgba(44, 127,255, 0.4);
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: beige;
     font: bold 14px;
     min-width: 4em;
     padding: 6px;
}

QPushButton:hover {
     background-color: rgba(255, 39, 107, 0.6);
}</string>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <zorder>lblImage</zorder>
   <zorder>gbResult</zorder>
   <zorder>lblQuestion</zorder>
   <zorder>lblCurrentStage</zorder>
   <zorder>lblLogo</zorder>
   <zorder>pbExit</zorder>
   <zorder>pbModule_2</zorder>
   <zorder>pbModule_1</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1364</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
